public/stylesheets/site/2.0/10-types-groups.css

```diff
@@ -190,12 +190,19 @@ span.series .position + .divider:after {
 }
 
 p.kudos {
-  background: url("/images/imageset.png") no-repeat -111px -580px;
-  padding: 0.5em 0.5em 0.5em 60px;
+  padding: 0.5em;
   margin: 2em 0;
   min-height: 40px;
 }
 
+p.kudos::before {
+  content: "";
+  background: url("/images/imageset.png") no-repeat -111px -580px;
+  width: 55px;
+  height: 54px;
```

Comment:
```suggestion
  background: url("/images/imageset.png") no-repeat -110px -580px;
  width: 55px;
  height: 55px;
```
This matches more what is in the png.

Reply:
![image](https://github.com/otwcode/otwarchive/assets/42606630/f7a58453-40a5-4adf-b1dd-bc286bb6e4a3)

It shows a bit of the adjacent images when viewed in Firefox if I change it to round numbers. Works fine on chrome though.